micro-async-data-writer/src/test/java/com/aol/micro/server/async/data/writer/MultiDataWriterTest.java

```diff
@@ -31,32 +32,33 @@ public void bothDataWritersUpdated() {
         writer.saveAndIncrement("hello world");
         assertThat(dataWriter1.loadAndGet()
                               .get(),
-                   equalTo("hello world"));
+                   equalTo(Try.success("hello world")));
         assertThat(dataWriter2.loadAndGet()
                               .get(),
-                   equalTo("hello world"));
+                   equalTo(Try.success("hello world")));
     }
 
     @Test
     public void emptySaveAndIncrement() {
+
         assertThat(empty.saveAndIncrement("hello world")
                         .get(),
-                   equalTo(null));
+                   equalTo(Try.success(null)));
     }
 
     @Test
     public void loadAndGetReturnsDataFromFirst() {
         dataWriter1.setData("one");
         dataWriter2.setData("two");
         String data = writer.loadAndGet()
-                            .get();
+                         .orElse(null);
         assertThat(data, equalTo("one"));
     }
 
     @Test
     public void loadAndGetReturnsNullForEmpty() {
         String data = empty.loadAndGet()
-                           .get();
+                           .orElse(null);
```

Comment:
maybe the ```orElse``` should return something else except null

Reply:
Um.. yes it should.. good catch!